### PR TITLE
Update local development instructions in README

### PR DESCRIPTION
**Pull request by ethomas2**

### CODE REVIEW

**Comment by zingales** [comment]
Did you install this on linux or on macOS? Also i think the create database isn't correct. Also the environment variables are saved in .env which is loaded by `heroku local` command.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,28 @@
 # Installation
 ## Locally
 * postgres server installed and running
-	* `brew install postgres`
-	* `pg_ctl -D /usr/local/var/postgres start`
-* heroku cli installed
-	* `brew install heroku`
+    * `brew install postgres`
+    * `pg_ctl start -D /usr/local/var/postgres`
```
Comment (zingales) [inline README.md:5]: Did the previous version not work on your machine?

File: README.md [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
-	* `brew install go`
-* godep installed: 
-	* `go get github.com/tools/godep`
+    * `brew install go`
+* godep installed:
+    * `go get github.com/tools/godep`
+* create database
+    * `psql -d postgres -c "create database cerealnotes_test"`
+    * Refer to `migrations/README.md` to create relevant database tables
+* export environment variables
```
Comment (zingales) [inline README.md:13]: If you used `heroku local` you wouldn't need to do this

File: README.md [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
+    ```
+    export DATABASE_URL=postgres://localhost:5432/cerealnotes_test?sslmode=disable
+    export TOKEN_SIGNING_KEY=abcdefg
+    export PORT=8000
+    ```
+* run the app
+    * `go build && ./cerealnotes`
```
Comment (zingales) [inline README.md:20]: why this instead of `go install && heroku local`?

File: README.md [hunk 4]
```diff
@@ -1,18 +1,28 @@
...
 
 ## Heroku
+* heroku cli installed
```
Comment (zingales) [inline README.md:23]: this needs to be installed to run locally

**Review by zingales — CHANGES_REQUESTED** [review]

**Comment by ethomas2** [comment]
* installed on macos.

* I just made the sepereate db because I found it cleaner. I usually keep a database per project/app. So I made a db specifically for cerealnotes so as not to clutter the default db with cerealnotes specific tables.

* I didn't notice `.env`. You're right that I would've saved a lot of time if I just used `heroku local` like the README told me to. I'm having trouble finding reasons why I was averse to using `heroku local`. G and I chatted offline about this, he told me to use `heroku local` to minimize the differences between local development and prod. Makes sense

File: README.md (hunk 1 above)
Comment (ethomas2) [reply README.md:5]: nope

```
mbp-IT002597:~ ethomas$ pg_ctl --version
pg_ctl (PostgreSQL) 10.3
mbp-IT002597:~ ethomas$ pg_ctl --help | head -10
pg_ctl is a utility to initialize, start, stop, or control a PostgreSQL server.

Usage:
  pg_ctl init[db] [-D DATADIR] [-s] [-o OPTIONS]
  pg_ctl start    [-D DATADIR] [-l FILENAME] [-W] [-t SECS] [-s]
                  [-o OPTIONS] [-p PATH] [-c]
  pg_ctl stop     [-D DATADIR] [-m SHUTDOWN-MODE] [-W] [-t SECS] [-s]
  pg_ctl restart  [-D DATADIR] [-m SHUTDOWN-MODE] [-W] [-t SECS] [-s]
                  [-o OPTIONS] [-c]
  pg_ctl reload   [-D DATADIR] [-s]
```

I tried checking the pg_ctl release notes to see if something changed. Couldn't find anything. But putting the optional args before `start` resulted in an error for me

File: README.md (hunk 3 above)
Comment (ethomas2) [reply README.md:20]: just to avoid heroku

**Comment by ethomas2** [comment]
closing in favor of https://github.com/atmiguel/cerealnotes/pull/39

**Pull request closed without merging**